README.md

```diff
@@ -11,6 +11,6 @@ This is the list of the [GitHub](https://github.com) usernames of all the people
 - [sparalic](https://github.com/sparalic/)
 - [dorothykiz1](https://github.com/dorothykiz1/)
 - [dujm](https://github.com/dujm)
-
+- [lusmoura](https://github.com/lusmoura)
```

Comment:
Thanks @lusmoura!

Not sure if you realized, but you've got this change from the other PR (that was expected). Let me explain what happened.

When you did the first changes, you did them in master, the only branch you had. Then, you created this new branch from that modified master where you added your username. And the new branch contains them, which is not what you wanted, because this changes are unrelated.

The take away here, is that you should keep your local master clean, and always work in other branches. So, you can always create branch from the code in the original repository.

If you want to practice you can start again, otherwise I can merge this. Up to you, but wanted you to learn this.